src/js/photo-upload.js

```diff
@@ -53,7 +53,7 @@ const loadExistingPhotos = async (photos) => {
     });
   }
   uppy.getFiles().forEach(file => {
-    console.log(file.id);
+    // console.log(file.id);
```

Comment:
I think we won't like to leave commented code if not necessary.

Reply:
:stuck_out_tongue: